numpy/lib/polynomial.py

```diff
@@ -1059,6 +1059,16 @@ def roots(self):
         """ The roots of the polynomial, where self(x) == 0 """
         return roots(self._coeffs)
 
+    # our internal _coeffs property need to be backed by __dict__['coeffs'] for
+    # scipy to work correctly. Note that as a result, the getter for .coeffs
+    # does not run unless accessed through one of its aliases.
```

Comment:
Sorry to have been out of the loop, but indeed the second sentence is incorrect.